docs/HowToUseIotivity.md

```diff
@@ -0,0 +1,12 @@
+How to use IoTivity
+===================
+
+IoTivity is an open source software framework enabling seamless device-to-device connectivity to address the emerging needs of the Internet of Things.
+If you want to know more detailed information, please find it at [README](../external/iotivity/iotivity_1.2-rel/README.md) and https://iotivity.org
```

Comment:
1.  how about below?
```
Please find details at README (link) and Iotivity site (link).
```
I heard we had better not use "you" on this kind of doc.
2. If you want to split a line from above, you shoudl add two spaces at end of above line.

Reply:
Ok